dotCMS/src/main/java/com/dotcms/ai/util/ContentToStringUtil.java

```diff
@@ -163,12 +164,19 @@ public List<Field> guessWhatFieldsToIndex(@NotNull Contentlet contentlet) {
             return List.of();
         }
 
-        final String ignoreUrlMapFields = (contentlet.getContentType().urlMapPattern() != null) ? contentlet.getContentType().urlMapPattern() : "";
+        final String ignoreUrlMapFields = org.apache.commons.lang3.StringUtils.defaultIfBlank(
+                contentlet.getContentType().urlMapPattern(),
+                StringPool.BLANK);
 
         contentlet.getContentType()
                 .fields()
-                .stream().filter(f -> !ignoreUrlMapFields.contains("{" + f.variable() + "}"))
-                .filter(f -> f instanceof StoryBlockField || f instanceof WysiwygField || f instanceof BinaryField ||  f instanceof TextAreaField || f instanceof FileField)
+                .stream()
+                .filter(f -> !ignoreUrlMapFields.contains("{" + f.variable() + "}"))
+                .filter(f -> f instanceof StoryBlockField
+                        || f instanceof WysiwygField
+                        || f instanceof BinaryField
+                        || f instanceof TextAreaField
+                        || f instanceof FileField)
```

Comment:
These are appreciated but again let's not include them as part of another chance because it makes difficult to understand/review the actual changes and inflates the PR.

Reply:
Rollbacked.